backends/qualcomm/builders/op_prelu.py

```diff
@@ -43,6 +43,10 @@ def define_node(
             coeff_node = node.args[1]
             coeff_tensor = torch.zeros(input_node.meta["val"].shape)
             coeff = get_parameter(coeff_node, self.edge_program)
+            # param nodes will be FakeTensor when doing partition
+            # fill in random numeric for validation
+            if isinstance(coeff, torch._subclasses.fake_tensor.FakeTensor):
+                coeff = torch.ones(coeff.shape)
```

Comment:
It's inserted, but we make it static inside the graph for it could be identified when building operator. This can reduce extra memory copies, I think.